metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala

```diff
@@ -64,64 +67,87 @@ case class ScalafixProvider(
 
   def organizeImports(
       file: AbsolutePath,
-      scalaTarget: ScalaTarget
+      scalaTarget: ScalaTarget,
+      retried: Boolean = false
   ): Future[List[l.TextEdit]] = {
     val fromDisk = file.toInput
     val inBuffers = file.toInputFromBuffers(buffers)
-    if (isUnsaved(inBuffers.text, fromDisk.text)) {
```

Comment:
This is mostly just moved, I recommend turning on the option to ignore whitespace.